packages/fxa-auth-server/fxa-oauth-server/test/routes/token.js

```diff
@@ -230,28 +228,33 @@ describe('/token POST', function() {
     });
 
     it('allows clients that have not been disabled via config', async () => {
+      request.payload.client_id = NON_DISABLED_CLIENT_ID;
```

Comment:
Does the description of this test match what is being tested? It seems more like "it fails if grant_type does not match passed in grant" or something similar.

Reply:
It does, but it's testing it in a kind of weird way in order to minimize setup; I'll add some explanation.